pkg/api/validation/validation.go

```diff
@@ -1761,7 +1761,11 @@ func ValidateVolumeMounts(mounts []api.VolumeMount, volumes sets.String, fldPath
 			allErrs = append(allErrs, field.Invalid(idxPath.Child("mountPath"), mnt.MountPath, "must be unique"))
 		}
 		if !path.IsAbs(mnt.MountPath) {
-			allErrs = append(allErrs, field.Invalid(idxPath.Child("mountPath"), mnt.MountPath, "must be an absolute path"))
+			// also allow windows absolute path
+			p := strings.ToUpper(mnt.MountPath)
```

Comment:
ToUpper can be costly since involves unicode lookup, also there's no need to convert remaining path to upper.

Consider just do compare literally:
https://github.com/golang/go/blob/b00b214e0d5eecfbb98a7a52d65da07a8c9628f7/src/os/path_windows.go#L60

Reply:
validation only run once, perf is not the problem here

Reply:
I think the concern is that it will run for _every_ pod, even if it only runs once.